k8s/collector-deployment.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: collector-db-creds
+  namespace: default
+type: Opaque
+data:
+  mysql-username: Y29sbGVjdG9ydXNlcg==
```

Comment:
We might have to come up with a better solution to checking in these values and regenerate the creds for the "real" DB.